awx/main/utils/licensing.py

```diff
@@ -221,8 +222,13 @@ def validate_rh(self, user, pw):
                     auth=(user, pw)
                 )
                 resp.raise_for_status()
-                json.extend(resp.json())
-
+                valid_pools = []
```

Comment:
This whole block shouldn't be needed. `480` filtering happens already via `generate_license_options_from_entitlements() --> is_appropriate_sub()`